src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -33,7 +34,7 @@ public class TypicalPersons {
             .withAddress("311, Clementi Ave 2, #02-25").withRoomNumber("05-0523")
             .withEmail("johnd@example.com").withPhone("98765432")
             .withEmergencyContact("Aiken", "98765431").withGradYear("2030")
-            .withTags("owesMoney", "friends").build();
+            .withTags("owes Money", "friends").build();
```

Comment:
is there mismatch with the VALID_TAG_OWES_MONEY: supposed to be "owes money"? or idk isit case sensitive?